gnome-project/config/settings.py

```diff
@@ -119,3 +119,7 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+
```

Comment:
what is this

Reply:
I'm guessing you're referring to the STATICFILES_DIRS variable. It's required if we want to use a single style sheet throughout the application.